cmd/gitter/main.go

```diff
@@ -197,13 +197,13 @@ func getCRDsFromTag(dir string, tag string, hash *plumbing.Hash, w *git.Worktree
 		return nil, err
 	}
 	reg := regexp.MustCompile("kind: CustomResourceDefinition")
-	regPath := regexp.MustCompile("^.*\\.yaml")
+	regPath := regexp.MustCompile(`^.*\.yaml`)
```

Comment:
@RichiCoder1 does this change have any functional difference? I'm not opposed to it, just wondering.

Reply:
Oops! Sorry, should have commented. My go linter was complaining about the unnecessary escape. Should be functionally the same